extension.xml

```diff
@@ -3,23 +3,27 @@
 <root version="3.0">
   <properties>
     <name>5E - Matjam's Status Indicators</name>
-    <version>1.0</version>
+    <version>1.3</version>
+    <!-- Please see the license.md file included with this distribution for attribution and copyright information. -->
+
+    <!-- Why in Lathander's name are you trying to load order this at 2000?! Because hook reasons-->
   	<loadorder>2000</loadorder>
     <author>matjam</author>
     <description>Show the status of a PC or NPC on the battlemap using graphical icon overlays.</description>
   </properties>
-
-  <announcement text="5E - Matjam's Save Indicators version 1.0" font="emotefont" icon="MatjamsExtension" />
+  <!-- Changed name from Save to Status on version 1.3 dated 7/2/20 -->
```

Comment:
doesn't need to be commented here; have the commit history and git blame if I need to figure it out.

Reply:
Is my Persuasion Check roll of 19 not enough?! Fine!